docs/AboutUs.md

```diff
@@ -2,58 +2,59 @@
 layout: page
 title: About Us
 ---
+# About InternBuddy
+We are team InternBuddy, and we are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
-
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `e0774066[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Christopher Tan Rui Yang
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/potty10.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/potty10)]
+[[portfolio](team/potty10.md)]
```

Comment:
Should we rename `portfolio` to `Personal Portfolio Page`? And we could capitalise all the links, such as `GitHub` instead of `github`.

Reply:
I don't think this is necessary. The requirement is to follow the provided About Us format closely.